tests/test_models/test_losses/test_ce_loss.py

```diff
@@ -86,4 +198,136 @@ def test_ce_loss():
         loss_name='loss_ce')
     loss_cls = build_loss(loss_cls_cfg)
     assert loss_cls.loss_name == 'loss_ce'
+
+    # test ce loss with ignore index
```

Comment:
also test ce loss with ignore index but `avg_non_ignore=False`

Reply:
When `avg_non_ignore=False`, `ignore index` should not  be set.